test/Interpreter/generic_casts.swift

```diff
@@ -215,135 +218,150 @@ func genericAnyAsUnconditional<T>(type: Any.Type, to: T.Type, expected: Bool) ->
 
 // CHECK-LABEL: casting types to protocols with generics:
 print("casting types to protocols with generics:")
-print(nongenericAnyIsPConforming(type: P.self)) // CHECK: false
-print(nongenericAnyIsPSubtype(type: P.self)) // CHECK: true
-print(genericAnyIs(type: P.self, to: P.self, expected: true)) // CHECK: true
-print(nongenericAnyIsPConforming(type: PS.self)) // CHECK: true
-print(genericAnyIs(type: PS.self, to: P.self, expected: true)) // CHECK-ONONE: true
-print(nongenericAnyIsPConforming(type: PE.self)) // CHECK: true
-print(genericAnyIs(type: PE.self, to: P.self, expected: true)) // CHECK-ONONE: true
-print(nongenericAnyIsPConforming(type: PC.self)) // CHECK: true
-print(genericAnyIs(type: PC.self, to: P.self, expected: true)) // CHECK-ONONE: true
-print(nongenericAnyIsPConforming(type: PCSub.self)) // CHECK: true
-print(genericAnyIs(type: PCSub.self, to: P.self, expected: true)) // CHECK-ONONE: true
+print(#line, nongenericAnyIsPType(type: P.self)) // CHECK: [[@LINE]] false
```

Comment:
Good move adding all these `#line`s. I hope you were able to automate that.

Reply:
Emacs macros FTW!